tests/tamr_client/dataset/test_dataset.py

```diff
@@ -30,3 +32,23 @@ def test_from_resource_id_dataset_not_found():
 
     with pytest.raises(tc.dataset.NotFound):
         tc.dataset.from_resource_id(s, instance, "1")
+
+
+@responses.activate
+def test_attributes():
+    s = fake.session()
+    dataset = fake.dataset()
+
+    attrs_url = replace(dataset.url, path=dataset.url.path + "/attributes")
+    attrs_json = utils.load_json("attributes.json")
+    responses.add(responses.GET, str(attrs_url), json=attrs_json, status=204)
```

Comment:
Let's include adding `ndjson` compatibility to `fake.json` so we can rewrite this without explicitly relying on `responses`.

Reply:
I think the `GET /dataset/{dataset_id}/attributes` endpoint returns a JSON list and not a stream.  I have taken a crack at adding `ndjson` compatibility to `fake.json`, but it doesn't seem to be applicable to the `attributes` functionality.  I could modify the testing for `record.stream` which would use this, but maybe that should be a separate PR.

Reply:
Ah good point. Yes, we should include `ndjson` capability in a PR that touches streaming. Let's leave out of this one.

Reply:
Note: we should still convert this test to use `fake.json`